apps/accounts/api/automations/change_secret.py

```diff
@@ -33,7 +33,7 @@ class ChangeSecretAutomationViewSet(OrgBulkModelViewSet):
     serializer_class = serializers.ChangeSecretAutomationSerializer
 
 
-class ChangeSecretRecordViewSet(mixins.ListModelMixin, OrgGenericViewSet):
+class ChangeSecretRecordViewSet(RecordListMixin, mixins.ListModelMixin, OrgGenericViewSet):
     filterset_class = ChangeSecretRecordFilterSet
     search_fields = ('asset__address', 'account__username')
     ordering_fields = ('date_finished',)
```

Comment:
Based on the given code snippet, no significant issues or differences have been detected. It appears to be correct according to the current knowledge cut-off (2021-09-01), with only minor modifications that should not affect its functionality. If you need any further clarification or additional testing, feel free to ask!